relayer/chains/cosmos/feegrant.go

```diff
@@ -299,72 +340,58 @@ func (cc *CosmosProvider) EnsureBasicGrants(ctx context.Context, memo string) (*
 
 		granterExists := cc.EnsureExists(cliCtx, granterAcc) == nil
 
-		//Feegranter exists on chain
+		// Feegranter exists on chain
 		if granterExists {
-			txResp, err := cc.SubmitTxAwaitResponse(ctx, msgs, memo, 0, granterKey)
+			txResp, err := cc.SubmitTxAwaitResponse(ctx, msgs, memo, gas, granterKey)
 			if err != nil {
-				fmt.Printf("Error: SubmitTxAwaitResponse: %s", err.Error())
 				return nil, err
 			} else if txResp != nil && txResp.TxResponse != nil && txResp.TxResponse.Code != 0 {
-				fmt.Printf("Submitting grants for granter %s failed. Code: %d, TX hash: %s\n", granterKey, txResp.TxResponse.Code, txResp.TxResponse.TxHash)
+				cc.log.Warn("Feegrant TX failed", zap.String("TX hash", txResp.TxResponse.TxHash), zap.Uint32("code", txResp.TxResponse.Code))
 				return nil, fmt.Errorf("could not configure feegrant for granter %s", granterKey)
 			}
 
-			fmt.Printf("TX succeeded, %d new grants configured, %d grants already in place. TX hash: %s\n", grantsNeeded, numGrantees-grantsNeeded, txResp.TxResponse.TxHash)
+			cc.log.Info("Feegrant succeeded", zap.Int("new grants", grantsNeeded), zap.Int("existing grants", numGrantees-grantsNeeded), zap.String("TX hash", txResp.TxResponse.TxHash))
```

Comment:
nit: log field names should be snake_case https://github.com/cosmos/relayer/blob/main/docs/logging.md#usage